Gemfile

```diff
@@ -5,10 +5,10 @@ require File.expand_path('../lib/regexp_extensions', FOREMAN_GEMFILE)
 
 source 'https://rubygems.org'
 
-gem 'rails', '3.2.21'
+gem 'rails', '4.0.12'
```

Comment:
I checked and Rails 4.1.5 will be in a future SCL, so we can target that for a release.  I agree with @eLobato that 4.0 doesn't seem a possible *release* target due to lack of security support - but we could upgrade to 4.0 and then to 4.1 before a release, if that's feasible.